newIDE/app/src/stories/componentStories/GameDashboard/Marketing/MarketingPlans.stories.js

```diff
@@ -0,0 +1,478 @@
+      'sl-SI':
+        'Popolno za ljudi z dokončano igro, ki jo želijo promovirati čim širši javnosti',
+      'uk-UA':
+        'Ідеально підходить для людей з готовою грою, які хочуть просувати її найбільш широкій аудиторії',
+      'zh-CN': '适合拥有完成游戏并希望将其推广给尽可能广泛的受众的人',
+    },
+    bulletPointsByLocale: [
+      {
+        en: 'Get a banner within GDevelop on desktop and the web editor',
+        'fr-FR': 'Obtenez une bannière dans GDevelop sur bureau et web',
+        'ar-SA': 'احصل على لافتة داخل GDevelop على سطح المكتب ومحرر الويب',
+        'de-DE':
+          'Erhalten Sie ein Banner in GDevelop auf Desktop und Web-Editor',
+        'es-ES':
+          'Obtenga un banner dentro de GDevelop en escritorio y el editor web',
+        'it-IT':
+          "Ottieni un banner all'interno di GDevelop su desktop e l'editor web",
+        'ja-JP': 'デスクトップとWebエディターのGDevelop内にバナーを表示します',
+        'ko-KR': '데스크톱 및 웹 편집기에서 GDevelop 내 배너 받기',
+        'pl-PL': 'Otrzymaj baner w GDevelop na komputerze i edytorze sieci web',
+        'pt-BR':
+          'Obtenha um banner dentro do GDevelop no desktop e no editor da web',
+        'ru-RU':
+          'Получите баннер в GDevelop на рабочем столе и в веб-редакторе',
+        'sl-SI':
+          'Pridobite pasico v GDevelop na namizju in spletnem urejevalniku',
+        'uk-UA':
+          'Отримайте банер у GDevelop на робочому столі та в веб-редакторі',
+        'zh-CN': '在桌面和Web编辑器中的GDevelop中获取横幅',
+      },
+      {
+        en: 'Hundreds of thousands of monthly users',
+        'fr-FR': "Des centaines de milliers d'utilisateurs mensuels",
+        'ar-SA': 'مئات الآلاف من المستخدمين الشهريين',
+        'de-DE': 'Hunderttausende von monatlichen Nutzern',
+        'es-ES': 'Cientos de miles de usuarios mensuales',
+        'it-IT': 'Centinaia di migliaia di utenti mensili',
+        'ja-JP': '数十万人の月間ユーザー',
+        'ko-KR': '수십만 명의 월간 사용자',
+        'pl-PL': 'Setki tysięcy użytkowników miesięcznie',
+        'pt-BR': 'Centenas de milhares de usuários mensais',
+        'ru-RU': 'Сотни тысяч ежемесячных пользователей',
+        'sl-SI': 'Stotine tisoč mesečnih uporabnikov',
+        'uk-UA': 'Сотні тисяч щомісячних користувачів',
+        'zh-CN': '数十万月度用户',
+      },
+    ],
+  },
+];
+
+export const LoadingAndError = () => {
+  return (
+    <AuthenticatedUserContext.Provider
+      value={fakeAuthenticatedUserWithNoSubscriptionAndCredits}
+    >
+      <MarketingPlans game={fakeGame} />
+    </AuthenticatedUserContext.Provider>
+  );
+};
+
+export const Default = () => {
+  return (
+    <AuthenticatedUserContext.Provider
+      value={fakeAuthenticatedUserWithNoSubscriptionAndCredits}
+    >
+      <MarketingPlans game={fakeGame} />
+    </AuthenticatedUserContext.Provider>
+  );
+};
+Default.parameters = {
+  mockData: [
+    {
+      url: `${
+        GDevelopGameApi.baseUrl
+      }/game-featuring?userId=indie-user&gameId=complete-game-id`,
```

Comment:
Should you get rid of the query params?

Reply:
for some reason the mock only works when the specific parameters are specified. Maybe that's an issue with the upgrade of storybook?